doc/ForcedUpdate.md

```diff
@@ -0,0 +1,22 @@
+# Forced Update
+
```

Comment:
Explain what the feature is in a few sentences before going into how it's implemented.